pkg/providers/tinkerbell/create.go

```diff
@@ -78,13 +78,14 @@ func (p *Provider) PostWorkloadInit(ctx context.Context, cluster *types.Cluster,
 
 	err := p.stackInstaller.Install(
 		ctx,
-		clusterSpec.VersionsBundle.Tinkerbell.TinkerbellStack,
+		clusterSpec.VersionsBundle.Tinkerbell,
```

Comment:
Looking at Install() below, wonder why this change was really needed. But no biggie either way.